README.md

```diff
@@ -28,10 +28,17 @@ gulp.task('default', () =>
 
 ## API
 
-### gulpRemark()
+### gulpRemark([options])
 
 Gulp plug-in. It processes your files through [remark][remark]. If you want to define specified options, use [`.remarkrc`][remarkrc]. The ignoring of files available using [`.remarkignore`][remarkignore] file.
 
+#### options
+
+Type: `Object`  
+Default: `{}`
```

Comment:
One can now pass both in the same object, `options.silent` *and* `options.commonmark`! So, why would that be necessary?